File: src/main/java/com/ject/studytrip/mission/infra/jpa/MissionJpaRepository.java
package com.ject.studytrip.mission.infra.jpa;

import com.ject.studytrip.mission.domain.model.Mission;
import java.util.List;
import org.springframework.data.jpa.repository.JpaRepository;

public interface MissionJpaRepository extends JpaRepository<Mission, Long> {
    List<Mission> findAllByIdIn(List<Long> ids);

    List<Mission> findAllByStampIdAndDeletedAtIsNullOrderByCreatedAt(Long stampId);
}

File: src/main/kotlin/com/ject/studytrip/mission/infra/jpa/MissionJpaRepository.kt
package com.ject.studytrip.mission.infra.jpa

import com.ject.studytrip.mission.domain.model.Mission
import org.springframework.data.jpa.repository.JpaRepository

interface MissionJpaRepository : JpaRepository<Mission, Long> {
    fun findAllByIdIn(missionIds: List<Long>): List<Mission>

    fun findAllByStampIdAndDeletedAtIsNullOrderByCreatedAt(stampId: Long): List<Mission>
}
